ssc/cmod_windpower.cpp

```diff
@@ -412,8 +464,12 @@ void cm_windpower::exec()
             throw exec_error("windpower", wpc.GetErrorDetails());
         }
 
-        if (wakeModelChoice != 3)
-            wake_int_loss_percent = (1. - farmPower / farmPowerGross) * 100.;
+        if (wakeModelChoice != CONSTANTVALUE) //we already threw an error if a wake loss multiplier was specified, so may just proceed
+        {
+            assign("annual_wake_loss_internal_kWh", var_data((ssc_number_t)(farmPowerGross - farmPower)));
```

Comment:
shouldn't the loss multiplier be applied here, similar to below?

```
            if (wakeModelChoice != CONSTANTVALUE && is_assigned("wake_loss_multiplier")) //wake loss multiplier isn't available for constant wake loss option
            {
                double wakeLossMultiplier = as_double("wake_loss_multiplier");
                double wakeLossBeforeMultiplier = gross_farmp - farmp;
                double newWakeLoss = wakeLossBeforeMultiplier * wakeLossMultiplier;
                farmp = gross_farmp - newWakeLoss;
            }
```

Reply:
Yup, you're right. Good catch!